spec/requests/cloud_volumes_spec.rb

```diff
@@ -131,8 +131,10 @@
 
   describe "safe delete" do
     it 'can safe delete cloud volumes which support safe_delete' do
-      ems = FactoryBot.create(:ems_autosde, :name => "Autosde")
-      volume = FactoryBot.create(:cloud_volume_autosde, :ext_management_system => ems, :name => "my_volume")
+      ems    = FactoryBot.create(:ext_management_system)
+      volume = FactoryBot.create(:cloud_volume, :ext_management_system => ems)
+      stub_supports(volume, :safe_delete)
```

Comment:
Better to use generic type and stub supports/supports_not than encode logic into the test based on what type is being used